### PR TITLE
[feat]: Adding css for cards

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added css for cards component.
1. Added a new image for card component.
2. Added a css file for same.

Below is the SS for same. Please review my PR.

Screenshot 2022-01-30 at 12 17 16 AM

### CODE REVIEW

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    box-sizing: border-box;
+    margin: 1.2rem;
+}
+
+.card-badge{
+    position: absolute;
+    top: 1.2rem;
+    background-color:#1FABD7;
+    color: white;
```
Comment (prachi-sahani) [inline components/cards/cards.css:18]: try using consistent units for colors

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    right: 10px;
+    color: white;
+    font-size: 1.2rem;
+}
+
+.card-dismiss:hover{
+    cursor: pointer;
+}
+
+.img-comp{
```
Comment (prachi-sahani) [inline components/cards/cards.css:34]: - you could have given a better name - something like card-img or img-header 
- also if you set top-border radius to image, it is breaking in one of the cards(the one which does not have description - 5th one)
   - you can set "overflow: hidden" in the parent container

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    width: 100%;
+    border-radius: 0.5rem 0.5rem 0 0;
+}
+
+
+
+.info-div:hover{
```
Comment (prachi-sahani) [inline components/cards/cards.css]: try using generic class names, something that shows what are you using it for.
for eg: card-info

File: components/cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.icons-comp{
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    margin-right: 1rem;
+}
+
+.icons-comp i{
```
Comment (prachi-sahani) [inline components/cards/cards.css]: do not use HTML tags(what if the user uses some other tag, maybe span tag?)

File: components/cards/cards.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    padding: 1rem;
+    color: gray;
+    font-size: 1.2rem;
+}
+
+i:hover{
```
Comment (prachi-sahani) [inline components/cards/cards.css]: do not use HTML tags

**Review by prachi-sahani — COMMENTED** [review]
Try using better naming convention, specially in case of component library. 
You can refer bootstrap for more clarity - like for lists they have used list-group for ul tag and list-group-item for it's li tags(it clearly shows the parent-child relation and what the tag is about)

reference - https://getbootstrap.com/docs/5.0/components/list-group/

**Review by ritik-patel05 — COMMENTED** [review]
Can use shorter or descriptive class names. 

Suggestion:
image => img
detail-comp => detail or detai-container
icons-comp => icons or icons-container

and others as suggested by @prachi-sahani

**Pull request merged**